flume/doc/design/architecture.md

```diff
@@ -1 +1,77 @@
-TBD
+#Cygnus architecture
```

Comment:
Current HA is done by replicating a complete Cygnus agent. There is nothing, internally, specifically addressing such an issue. All the architecture variants try to addresse performance issues.